### ISSUE TITLE
Infinite Grenades & Damage/Impact Screen Effect

### ISSUE THREAD

**Issue by ErrorXII** [Zet0rz/nzombies#636]
Occasionally, a seemingly infinite amount of grenades are given after buying a weapon from the wall or the mystery box which decrease as the weapon runs out of ammo but fill back up after buying ammo for the weapon.

Also, the red damage indicator needs to be toned down when getting hit from the front. It's as if a flashbang has gone off in its current state and shouldn't leave you blind and unable to escape, especially with juggernog.

**Comment by ghost** [comment]
I have no idea about the grenades as I have never had this happen to me personally, but you can get the Unreal Enhancements and the Chroma Color Correction addons by TFA off of the workshop to fix the blinding red-screen effect.

**Comment by ErrorXII** [comment]
Thanks for the recommendation. There are rounds where I spawn with no grenades at all and no amount of max ammos restore them which renders Widow useless if it's in the config.

There should probably be a whole separate issue opened for zombies executing their current action after death, mainly hitting me despite being a corpse on the ground already and breaking a barrier after being killed behind it.

**Comment by ghost** [comment]
@ErrorXII 
Yeah the bug where they still hit you after they die was fixed in a previous update, but it seems to have come back.

A surefire way to see the bug is get a very weak weapon on insta kill and kill a zombie that's right in front of you. It will ALWAYS hit you 1-2 times after it's been killed.

EX. 25 damage weapon on round 3, zombies have 90~ HP, get insta kill, zombies still hit you after they die.

100 damage weapon on round 2, zombies have less then 100 HP, get insta kill, zombies don't do anything after death except ragdoll.

It's weird how it works.
